src/utils.cpp

```diff
@@ -482,5 +495,310 @@ nlohmann::json getParsedJson(const std::string& pathToJson)
         throw std::runtime_error("Failed to parse JSON file");
     }
 }
+
+bool processDisableTag(const nlohmann::json& i_parsedConfigJson,
```

Comment:
API is handling only gpio related, function name should have gpio ?

Reply:
Could have been, but this was part of initial implementation.
I moved this under some other API. This is a dead code. Will remove the API itself.

Reply:
okay